src/glaciers/glacier/glacier2D_utils.jl

```diff
@@ -181,6 +184,13 @@ function init_gdirs(rgi_ids::Vector{String}, params::Parameters; velocities=true
     try
         gdirs::Vector{PyObject} = workflow.init_glacier_directories(rgi_ids)
         filter_missing_glaciers!(gdirs, params)
+
+        if params.OGGM.DEM_source != "Default"
```

Comment:
Cool. Could you please add a short comment explaining what you are doing here?

Reply:
Wrote the comment: # Set different surface topography source if specified.

Just for your info, why I work with this string 'Default': the default source for glaciers can be different depending on their region. 

I am not sure why and how this is, but I noticed with for example Region 03: Arctic Canada North and Region 11: Central Europe, that they used different sources as a default.

Reply:
Yes, high latitude glaciers use Arctic DEM instead of SRTM.